tools/check_classes_mlc.py

```diff
@@ -0,0 +1,58 @@
+import argparse
+import json
+import os
+
+
+parser = argparse.ArgumentParser(description='Sample showcasing the new API')
+parser.add_argument('data_path', help='path to the dataset root')
+parser.add_argument('--train_annotation',
+                    help='path to the pre-trained model weights',
+                    default='train.json')
+parser.add_argument('--val_annotation',
+                    help='path to the pre-trained model weights',
+                    default='val.json')
+parser.add_argument('--clean', action='store_true')
+args = parser.parse_args()
+
+all_classes = []
+root = args.data_path
+subsets=[args.val_annotation, args.train_annotation]
+
+for subset in subsets:
```

Comment:
Maybe is it better to wrap into the function?
and call under ___name___==___main___

Reply:
Yes, at least for consistency